src/topology/mod.rs

```diff
@@ -42,16 +42,25 @@ type BuiltBuffer = (
     buffers::Acker,
 );
 
+type Outputs = HashMap<String, fanout::ControlChannel>;
+
+// Watcher types for topology changes. These are currently specific to receiving
+// `Outputs`. This could be expanded in the future to send an enum of types if, for example,
+// this included a new 'Inputs' type.
+type WatchTx = watch::Sender<Outputs>;
+pub type WatchRx = watch::Receiver<Outputs>;
+
 #[allow(dead_code)]
 pub struct RunningTopology {
     inputs: HashMap<String, buffers::BufferInputCloner>,
-    outputs: HashMap<String, fanout::ControlChannel>,
+    pub outputs: Outputs,
```

Comment:
Does this still need to be `pub`?

Reply:
Ah, nope - I thought I changed that, but it snuck back in somewhere. Thanks for catching!

Reply:
Nixed `pub` in 19aa7cd7e1bd9ee4a91bd8140a5e1448e9f8a7b1.